### PR TITLE
add methods to graphql provider

### PR DESCRIPTION
**Pull request by tshedor**

### CODE REVIEW

File: packages/brick_graphql/lib/src/annotations/graphql_serializable.dart [hunk 1]
```diff
@@ -23,20 +21,54 @@ enum FieldRename {
...
+  final String? defaultDeleteOperation;
+
+  /// The query used to fetch multiple member.
+  /// Only the header of the operation is required. For example
+  /// ```graphql
+  /// query GetPeople() {
+  ///   getPerson() {}
+  /// }
+  /// ```
+  final String? defaultGetUnfilteredOperation;
```
Comment (Josiassejod1) [inline packages/brick_graphql/lib/src/annotations/graphql_serializable.dart:40]: defaultGetOperation in opposition to unfiltered, just cause it feels better

File: packages/brick_graphql/lib/src/graphql_provider.dart [hunk 2]
```diff
@@ -1,22 +1,60 @@
...
+      ),
+    );
+    final resp = await link.request(request).first;
+    if (resp.data == null) return [];
+    if (resp.data?.keys.first is Iterable) {
+      final results = resp.data?.values.first
+          .map((v) => adapter.fromGraphql(v, provider: this, repository: repository))
+          .toList()
+          .cast<Future<_Model>>();
+
```
Comment (Josiassejod1) [inline packages/brick_graphql/lib/src/graphql_provider.dart:48]: 👨🏾‍🍳👨🏾‍🍳👨🏾‍🍳👨🏾‍🍳👨🏾‍🍳👨🏾‍🍳

File: packages/brick_graphql/lib/src/annotations/graphql_serializable.dart [hunk 3]
```diff
@@ -23,20 +21,54 @@ enum FieldRename {
...
+  /// The mutation used to remove data.
+  /// Only the header of the operation is required. For example
+  /// ```graphql
+  /// mutation DeletePerson($input: DeletePersonInput!) {
+  ///   deletePerson(input: $input) {}
+  /// }
+  /// ```
+  final String? defaultDeleteOperation;
+
+  /// The query used to fetch multiple member.
```
Comment (devbrandonlong) [inline packages/brick_graphql/lib/src/annotations/graphql_serializable.dart:33]: nit: 
```suggestion
  /// The query used to fetch multiple members.
```

File: packages/brick_graphql/lib/src/annotations/graphql_serializable.dart (hunk 1 above)
Comment (devbrandonlong) [reply packages/brick_graphql/lib/src/annotations/graphql_serializable.dart:40]: Think I like this as well fits with the other basic CRUD string names

**Review by devbrandonlong — APPROVED** [review]
minor nits. this is going to be major for the community

**Review by mateominato — APPROVED** [review]

**Pull request merged**